backend/index.js

```diff
@@ -1,13 +1,29 @@
 const express = require('express')
 const db = require('./db')
+const axios = require('axios')
 
 const stats = {}
 const decks = {}
 const app = express()
+let cards
+axios
+  .get('https://api.hearthstonejson.com/v1/29933/enUS/cards.json')
+  .then(function (response) {
```

Comment:
Please use arrow functions for coding style consistency.
```suggestion
  .then(response => db.setCards(response.data))
```

Reply:
You mean lamda ? :)

Reply:
I mean [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).